build.sbt

```diff
@@ -1,6 +1,6 @@
 import scala.collection.JavaConverters._
 
-scalaVersion := "2.12.15"
+ThisBuild / scalaVersion := "2.12.15"
```

Comment:
`ThisBuild` is required here otherwise sbt-github-actions doesn't pick up the Scala version (hence why it was 2.12.14 previously in the `ci.yaml`.